### PR TITLE
Dev/iwu/us vbs22 40  actualizacion de usuario nuevo

### PR DESCRIPTION
**Pull request by ichivancrCenfotec**
Changed credentials

### CODE REVIEW

File: demo/src/main/java/com/project/demo/entity/LoginResponse.java [hunk 1]
```diff
@@ -9,6 +9,15 @@ public class LoginResponse {
 
     private long expiresIn;
 
+    private Integer user_id;
```
Comment (nefi-cenfo) [inline demo/src/main/java/com/project/demo/entity/LoginResponse.java]: Please consider to use camel case format.

File: demo/src/main/java/com/project/demo/rest/AuthRestController.java [hunk 2]
```diff
@@ -84,4 +88,38 @@ public ResponseEntity<?> registerUser(@RequestBody User user) {
         User savedUser = userRepository.save(user);
         return ResponseEntity.ok(savedUser);
     }
+
+    @PostMapping("/reset-password")
+    public ResponseEntity<?> resetPassword(@RequestBody User user) {
+        Optional<User> optionalUser = userRepository.findByEmail(user.getEmail());
```
Comment (nefi-cenfo) [inline demo/src/main/java/com/project/demo/rest/AuthRestController.java]: Please try to use a service layer instead.

File: demo/src/test/java/com/project/demo/repository/UserRepositoryTests.java [hunk 3]
```diff
@@ -125,60 +125,35 @@ public void infoSetup() {
...
+    public void findUserToResetPassword() {
+        Optional<User> userFoundOptional = userRepository.findByEmail("test@gmail.com");
         Assertions.assertThat(userFoundOptional).isPresent();
-        User userFound = userFoundOptional.get();
-        Assertions.assertThat(userFound).isNotNull();
-        Assertions.assertThat(userFound.getName()).isEqualTo("John");
-    }
-
-    @Test
-    public void testLoginSuccess() {
```
Comment (nefi-cenfo) [inline demo/src/test/java/com/project/demo/repository/UserRepositoryTests.java]: Watch out for deleting existing tests.

**Review by nefi-cenfo — DISMISSED** [review]

File: demo/src/main/java/com/project/demo/entity/LoginResponse.java (hunk 1 above)
Comment (mur-tz) [reply demo/src/main/java/com/project/demo/entity/LoginResponse.java]: Iván already refactored it. Fixing comment.

File: demo/src/main/java/com/project/demo/rest/AuthRestController.java (hunk 2 above)
Comment (mur-tz) [reply demo/src/main/java/com/project/demo/rest/AuthRestController.java]: Iván updated his class. He is now using the service layer to then interact with the repository.

File: demo/src/test/java/com/project/demo/repository/UserRepositoryTests.java (hunk 3 above)
Comment (mur-tz) [reply demo/src/test/java/com/project/demo/repository/UserRepositoryTests.java]: Iván added again the missing tests. Resolving comment.

**Review by mur-tz — APPROVED** [review]

**Review by DieS06 — APPROVED** [review]

**Pull request merged**